packages/spark-core/settings/_settings.scss

```diff
@@ -29,7 +29,7 @@ $space-inset-xl: 64px 64px 64px 64px !default;
 $space-inset-short-xs: 2px 4px 2px 4px !default;
 $space-inset-short-s: 4px 8px 4px 8px !default;
 $space-inset-short-m: 8px 16px 8px 16px !default;
-$space-inset-short-l: 16px 32px 16px 32px !default;
+$space-inset-short-l: 16px 24px 16px 24px !default;
```

Comment:
Why was this change made?

Reply:
Button padding was changed in the new design doc.

Reply:
We don't want to change this value. This is on the normal scale. You'll want to create a var for buttons. Something like $button-padding and set that to $space-m $sprk-space-misc-a; The misc spacing stuff is in my PR though lol.

Reply:
@yodasw16  this affects everything that is using that variable. So, we shouldn't move forward with this.

Reply:
What we can do in the meantime is this `$button-padding: $space-m 24px`